src/main/java/agh/ddd/groups/poll/sagas/PollDeadlineSaga.java

```diff
@@ -0,0 +1,62 @@
+package agh.ddd.groups.poll.sagas;
+
+import agh.ddd.groups.poll.events.PollCreatedEvent;
+import agh.ddd.groups.poll.events.PollDeadlineReachedEvent;
+import agh.ddd.groups.poll.events.PollFinishedEvent;
+import agh.ddd.groups.poll.events.PollProlongedEvent;
+import org.axonframework.eventhandling.scheduling.EventScheduler;
+import org.axonframework.eventhandling.scheduling.ScheduleToken;
+import org.axonframework.saga.annotation.AbstractAnnotatedSaga;
+import org.axonframework.saga.annotation.SagaEventHandler;
+import org.axonframework.saga.annotation.StartSaga;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.io.Serializable;
+
+/**
+ * @author Michał Ciołczyk
+ */
+@Component
+public class PollDeadlineSaga extends AbstractAnnotatedSaga implements Serializable {
```

Comment:
I don't know exactly how to test this. Based on https://github.com/lc-nyovchev/axon-simple/blob/master/src/main/java/org/athend/axon/simple/domain/UserLockSaga.java